src/garage/experiment/experiment.py

```diff
@@ -1,158 +1,21 @@
+"""Tools for running experiments with Garage."""
```

Comment:
this file contains so much nonsense which we're trying to delete. can you put your new stuff into a different module so it's clear what's deprecated and what's not?

i'm even fine with you renaming this `experiment_deprecated` or whatever

Reply:
That would be good to do at some point soon. Especially once people have gotten used to `wrap_experiment`.